### PR TITLE
adding napalm rollback

### PR DESCRIPTION
**Pull request by jchristopher327**
Moving from the nornir repo to this one as suggested by Kirk.

https://github.com/nornir-automation/nornir/pull/552

### CODE REVIEW

File: tests/unit/test_napalm_rollback.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import os
+
+from nornir.plugins.tasks import networking
```
Comment (dbarrosop) [inline tests/unit/test_napalm_rollback.py:3]: wrong import path

File: tests/unit/test_napalm_rollback.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+
+THIS_DIR = os.path.dirname(os.path.realpath(__file__)) + "/mocked/napalm_rollback"
```
Comment (dbarrosop) [inline tests/unit/test_napalm_rollback.py:6]: I think you may have forgotten to add the test files

**Comment by dbarrosop** [comment]
You also need to add it here: https://github.com/nornir-automation/nornir_napalm/blob/master/nornir_napalm/plugins/tasks/__init__.py

**Comment by dbarrosop** [comment]
Hi! Thanks for the PR, I dropped a few comments.

**Comment by jchristopher327** [comment]
Thanks, I resolved your comments. I'm struggling with what the tests and what should be returned when a rollback is performed. I think a diff is what should be returned but not 100% sure. Any guidance would be appreciated.

**Comment by ktbyers** [comment]
Superseded by:

https://github.com/nornir-automation/nornir_napalm/pull/45

**Pull request closed without merging**